tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleTooling.java

```diff
@@ -99,17 +61,24 @@ else if( !bundleLocationString.contains( installPath ) )
 
             final ModelBuilder<T> modelBuilder = (ModelBuilder<T>) connection.model( modelClass );
 
+            final File depsDir = new File(cacheDir, "deps");
```

Comment:
I'll go ahead and merge this now but send me another pull with the new cache changes.